grails-app/conf/UrlMappings.groovy

```diff
@@ -38,12 +38,14 @@ class UrlMappings {
 		"/api/v1/canvases/$id/stop"(controller: "canvasApi", action: "stop")
 		"/api/v1/canvases/$resourceId/permissions"(resources: "permissionApi", excludes: ["create", "edit", "update"]) { resourceClass = Canvas }
 		"/api/v1/canvases/$resourceId/permissions/me"(controller: "permissionApi", action: "getOwnPermissions") { resourceClass = Canvas }
+		"/api/v1/canvases/$resourceId/permissions/remove/$id"(controller: "permissionApi", action: "remove") { resourceClass = Canvas }
```

Comment:
This change is unnecessary: The correct API call should be `DELETE /api/v1/.../permissions/$id`, which is already auto-mapped to the `PermissionApiController#delete` action by line 39. 

The `delete` action should support this use cases instead of adding a `remove` action. We should not need to introduce a new API endpoint or action for this.

Reply:
@hpihkala In the current code I'm not calling canShare() at all. Is this correct?